lib/java/build.gradle

```diff
@@ -0,0 +1,495 @@
+
+    systemProperties = [
+      'javax.net.ssl.trustStore': "${projectDir}/test/.truststore",
+      'javax.net.ssl.trustStorePassword': 'thrift'
+    ]
+
+    if (project.hasProperty('testargs')) {
+        args "$testargs".split(' ')
+    }
+}
+
+task testserver(type: JavaExec, group: 'Validation') {
+    description="Run a test server"
+    classpath = sourceSets.test.runtimeClasspath
+    main 'org.apache.thrift.test.TestServer'
+
+    systemProperties = [
+      'javax.net.ssl.keyStore': "${projectDir}/test/.keystore",
+      'javax.net.ssl.keyStorePassword': 'thrift'
+    ]
+
+    if (project.hasProperty('testargs')) {
+        args "$testargs".split(' ')
+    }
+}
+
+task testnonblockingserver(type: JavaExec, group: 'Validation') {
+    description="Run a test nonblocking server"
+    classpath = sourceSets.test.runtimeClasspath
+    main 'org.apache.thrift.test.TestNonblockingServer'
+
+    systemProperties = [
+      'javax.net.ssl.keyStore': "${projectDir}/test/.keystore",
+      'javax.net.ssl.keyStorePassword': 'thrift'
+    ]
+
+    if (project.hasProperty('testargs')) {
+        args "$testargs".split(' ')
+    }
+}
+
+// Main Unit Test task configuration
+test {
+    description="Run the full test suite"
+    dependsOn deprecatedEqualityTest, deprecatedJavaBeansTest
+
+    // Allow repeating tests even after successful execution
+    if (project.hasProperty('rerunTests')) {
+        outputs.upToDateWhen { false }
+    }
+
+    include '**/Test*.class'
+    exclude '**/Test*\$*.class'
+    exclude '**/TestClient.class'
+    exclude '**/TestServer.class'
+    exclude '**/TestNonblockingServer.class'
+
+    maxHeapSize = '512m'
+    forkEvery = 1
+
+    systemProperties = [
+      'build.test': "${compileTestJava.destinationDir}",
+      'test.port': "${testPort}",
+      'javax.net.ssl.trustStore': "${projectDir}/test/.truststore",
+      'javax.net.ssl.trustStorePassword': 'thrift',
+      'javax.net.ssl.keyStore': "${projectDir}/test/.keystore",
+      'javax.net.ssl.keyStorePassword': 'thrift'
+    ]
+}
+
+// ----------------------------------------------------------------------------
+// Allow this configuration to be shared between install and uploadArchives tasks
+def configurePom(pom) {
+    pom.project {
+        name 'Apache Thrift'
+        description 'Thrift is a software framework for scalable cross-language services development.'
+        packaging 'jar'
+        url 'http://thrift.apache.org'
+
+        scm {
+            url 'https://git-wip-us.apache.org/repos/asf?p=thrift.git'
+            connection 'scm:git:https://git-wip-us.apache.org/repos/asf/thrift.git'
+            developerConnection 'scm:git:https://git-wip-us.apache.org/repos/asf/thrift.git'
+        }
+
+        licenses {
+            license {
+                name 'The Apache Software License, Version 2.0'
+                url "${project.license}"
+                distribution 'repo'
+            }
+        }
+
+        developers {
```

Comment:
This list can be condensed down to:
```
developer {
  id 'dev'
  name 'Apache Thrift Developers'
  email 'dev@thrift.apache.org'
}
```

I believe this information is used elsewhere, for example in CPAN packaging.

Reply:
Sure, that would make the pom file much shorter and less likely to have stale information.